lambda/events_id/delete/index.js

```diff
@@ -37,51 +43,51 @@ exports.handler = async (event, context, callback) => {
 		callback(JSON.stringify(response), null);
 	
     }
-
-
-   // determine what http request is made to handle it appropriately 
-   // *** only using DELETE now but will need the others soon
-   if (httpMethod === 'DELETE') {
-   	
-		// params contains the table name and primary key to delete an item with the given url parameter
-	    var params = {
-	        TableName : "ak-api-dynamo",
-	        Key : {
-				"event_id" : ID
-			},
-			// needed to return deleted item or empty object if nothing found to delete by given ID
-	        "ReturnValues": "ALL_OLD"
-		};
-		
-
-		try {
-			// delete method is needed to delete items from a Dynamodb table
-		    const data = await documentClient.delete(params).promise();
-		    
-		    // check if nothing was returned to cause a 404 response, item not found in db 
-		    if(Object.entries(data).length === 0 && data.constructor === Object) {
-		    	// log needed for when a parameter eventID is not found in the db
-		    	console.log(`event_id ${ID} not found in Table.`);
-		    	response.statusCode = 404;
-		    	callback(JSON.stringify(response), null);
-		    }
-		    else {
-		    	// log needed to confirm successful deletion of item in db
-		    	console.log("Item successfully deleted.");
-		    	// response.statusCode = 200; already set to 200
-		    	callback(null, "Item Deleted");
-		    }
-			 
-		} catch(err) {
-			// error log needed to record internal server errors
-			console.error("Internal Server Error: ", err);
-			response.statusCode = 500;
-			callback(JSON.stringify(response), null);
-		}
-		
-   }
-   
-
+    else {
+	
+	   // determine what http request is made to handle it appropriately 
+	   // *** only using DELETE now but will need the others soon
+	   if (httpMethod === 'DELETE') {
```

Comment:
DELETE is only method that can trigger this Lambda (because the Gateway is handling the routing), so this conditional is not necessary.

Reply:
Thanks, I removed the http if block.